nixos/modules/services/network-filesystems/kbfs.nix

```diff
@@ -48,6 +48,7 @@ in {
       requires = [ "keybase.service" ];
       after = [ "keybase.service" ];
       path = [ "/run/wrappers" ];
+      unitConfig.ConditionUser = "!@system";
```

Comment:
Can you please explain why?

Reply:
I think this only in here because of a funky rebase that brings in lot of unrelated commits.

Reply:
That is, this was rebased on master, which has a bunch of commits including feed9154c0ccd8b42b46cc7ddcc65e46fb08eed4 that are not on staging yet.